borg/testsuite/helpers.py

```diff
@@ -160,70 +160,105 @@ def test(self):
         )
 
 
-def check_patterns(files, paths, excludes, expected):
-    """Utility for testing exclusion patterns.
+def check_patterns(files, pattern, expected):
+    """Utility for testing patterns.
     """
-    patterns = adjust_patterns(paths, excludes)
-    included = [path for path in files if not exclude_path(path, patterns)]
+    assert all([i == os.path.normpath(i) for i in files]), \
```

Comment:
maybe rather use f for the file, not i.

i somehow rather feels like an integer counter.